src/super_gradients/training/transforms/transforms.py

```diff
@@ -538,8 +538,8 @@ def __call__(self, sample: dict):
             origin_img, origin_labels = sample["image"], sample["target"]
             cp_sample = sample["additional_samples"][0]
             img, cp_labels = cp_sample["image"], cp_sample["target"]
-
-            img, cp_labels = _mirror(img, cp_labels, 0.5)
+            cp_boxes = cp_labels[:, :4]
+            img, cp_boxes = _mirror(img, cp_boxes, 0.5)
             jit_factor = random.uniform(*self.mixup_scale)
```

Comment:
Hm, I wonder if it would be correct to take a flip prob from dataset params for mixup as well. It matches anyway, and if it won't (e.g. I override flip_prob with 0.), then arguably the same flipping prob should be used in mixup

Reply:
Done. Passing now flip_prob to mixup as well.